rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/Config.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.connect.jdbc;
+
+import io.openmessaging.KeyValue;
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+import javax.jms.Session;
+
+public class Config {
+    @SuppressWarnings("serial")
+
+    public String jdbcAddr;
+    public Integer jdbcPort;
+    public String jdbcUsername;
+    public String jdbcPassword;
+
+    public String queueName;
+
+    public static final Set<String> REQUEST_CONFIG = new HashSet<String>() {
+        {
+            add("jdbcAddr");
+            add("jdbcPort");
+            add("jdbcUsername");
+            add("jdbcPassword");
+        }
+    };
+
+//    public String destinationType;
```

Comment:
It seems that this file copied from jms connector, and please remove these Irrelevant code